test/blackbox-tests/test-cases/named-targets/named-targets.t

```diff
@@ -0,0 +1,20 @@
+# Basic test for multiple targets
+  $ echo '(lang dune 3.8)' > dune-project
+  $ cat > dune << 'EOF'
+  > (rule
+  >  (targets output.txt secondary.log)
```

Comment:
How does this test the named targets feature? Neither of these targets are named.